hpx/parallel/algorithm.hpp

```diff
@@ -38,5 +39,7 @@
 #include <hpx/parallel/algorithms/set_union.hpp>
 #include <hpx/parallel/algorithms/sort.hpp>
 #include <hpx/parallel/algorithms/swap_ranges.hpp>
+#include <hpx/parallel/algorithms/uninitialized_copy.hpp>
+#include <hpx/parallel/algorithms/uninitialized_fill.hpp>
```

Comment:
We have those in `hpx/parallel/memory.hpp`, just like the original `uninitialized_xxx` algorithms. The TS is not very specific on what algorithms should be defined by which header, thus we decided to stay as close to the sequential algorithms.